### PR TITLE
Bug 1289726 - Fix subtest compare view window title and header

### PR DESCRIPTION
**Pull request by rwood-moz**
This change is [Reviewable](https://reviewable.io/reviews/mozilla/treeherder/1747)

### CODE REVIEW

File: ui/js/controllers/perf/compare.js [hunk 1]
```diff
@@ -413,15 +413,14 @@ perf.controller('CompareSubtestResultsCtrl', [
             $scope.compareResults = {};
             $scope.titles = {};
 
-            $scope.subtestTitle = ($scope.platformList[0].split(' ')[0] + " " +
-                                   $scope.testList[0].split(' ')[0]);
-            window.document.title = $scope.subtestTitle + " subtest comparison";
```
Comment (jmaher) [inline ui/js/controllers/perf/compare.js]: we are removing 'subtest comparison', is it clear from the rest of the page that we are looking at subtests?

Comment (rwood-moz) [inline ui/js/controllers/perf/compare.js]: Yes, on the page header itself, for example is: "linux64: basic_compositor_video pgo subtest summary". I can add 'subtest comparison' or 'summary' back to the window title too if you like!

File: ui/js/controllers/perf/compare.js [hunk 2]
```diff
@@ -413,15 +413,14 @@ perf.controller('CompareSubtestResultsCtrl', [
...
-
             var testName = $scope.testList[0].replace('summary ', '');
 
             $scope.titles[testName] = $scope.platformList[0] + ': ' + testName;
             $scope.compareResults[testName] = [];
 
+            window.document.title = $scope.titles[testName];
+            $scope.subtestTitle = $scope.titles[testName];
```
Comment (wlach) [inline ui/js/controllers/perf/compare.js]: Not a big deal, but how about:

```js
window.document.title = $scope.subtestTitle = $scope.titles[testName];
```

?

Comment (rwood-moz) [inline ui/js/controllers/perf/compare.js]: Right, thanks

File: ui/js/controllers/perf/compare.js (hunk 1 above)
Comment (jmaher) [inline ui/js/controllers/perf/compare.js]: no need to modify anything here per my original comment/question.

**Pull request merged**